crates/core_simd/src/libmf32.rs

```diff
@@ -0,0 +1,35 @@
+use crate::SimdF32;
+
+impl<const LANES: usize> SimdF32<LANES>
+where
+    Self: crate::LanesAtMost32,
+{
+    // This does not seem to be implemented, yet.
+    #[inline]
+    fn mul_add(self, mul: Self, add: Self) -> Self {
+        self * mul + add
+    }
```

Comment:
```suggestion
``` 
A hardware-accelerable version of this was implemented in #138.

Reply:
Is that version compiled to libm calls when hardware support isn't guaranteed to be available? Feels like a dubious tradeoff for this if so...

Reply:
Yes, because it is IEEE-754 compliant.

In general since we have a path forward for fixing the libm problem in LLVM we shouldn't get hung up on it elsewhere.